app/src/main/java/org/oppia/android/app/devoptions/ForceCrashListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.app.devoptions
+
+/** Listener for force crashing the app. */
+interface ForceCrashListener {
+  fun forceCrash()
```

Comment:
This should also be documented, e.g. ``Called when the 'force crash' button is clicked by the user.``

Reply:
Done.